cliff.toml

```
@@ -0,0 +1,5 @@
+# git-cliff ~ configuration file
+# https://git-cliff.org/docs/configuration
+
+[git]
+conventional_commits = false
```

Would suggest that we still add this file to ensure that everyone uses the same config, but with conventional_commits = true

```suggestion
conventional_commits = true
```